tokio/src/macros/cfg.rs

```diff
@@ -354,6 +354,15 @@ macro_rules! cfg_uds {
     }
 }
 
+macro_rules! cfg_unix {
+    ($($item:item)*) => {
+        $(
+            #[cfg(unix)]
+            $item
+        )*
+    }
+}
```

Comment:
We use `#[cfg(unix)]` everywhere else in the codebase. Maybe adding this `cfg_*` makes sense, but I would rather want it in its own PR.

Reply:
[Removed](https://github.com/tokio-rs/tokio/pull/2524/commits/384d1850e74d529666871a7008e44724b9ea6a3b).